Tests/SwiftGenKitTests/CatalogEntryTests.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  CatalogEntryTests.swift
+//  Tests
+//
+//  Created by Francisco Diaz on 10/5/17.
+//  Copyright © 2017 SwiftGen. All rights reserved.
+//
+
+import XCTest
+@testable import SwiftGenKit
+
+final class CatalogEntryTests: XCTestCase {
+  func testGroupEntry_IsComparableAlphabetically() {
+    let entryA = Catalog.Entry.group(name: "A", items: [])
+    let entryB = Catalog.Entry.group(name: "B", items: [])
```

Comment:
Could be nice to also add some tests using different letter case to check if that comparability is case-sensitive or not (but whether it is or not, that it always sorts `["A","a"].map { Catalog.Entry.group(name: $0, items: []) }` in the same order

Reply:
Now that I think about it, I should also test that two groups with the same name order its items correctly.

I'll work on that 👍